hydrolib/core/basemodel.py

```diff
@@ -778,6 +784,18 @@ def _relative_mode(self) -> ResolveRelativeMode:
         """Gets the ResolveRelativeMode of this FileModel.
 
         Returns:
-            ResolveRelativeMode: The ResolveRelativ
+            ResolveRelativeMode: The ResolveRelativeMode of this FileModel
+        """
+        return ResolveRelativeMode.ToParent
+
+    @classmethod
+    def _get_relative_mode_from_data(cls, data: Dict[str, Any]) -> ResolveRelativeMode:
+        """Gets the ResolveRelativeMode of this FileModel based on the provided data.
```

Comment:
I recommend documenting in this most general FileModel implementation of this function that it actually does *not* use the input `data` for getting the ResolveRelativeMode, as the base case that can be overridden by subclasses.